psdk_wrapper/src/modules/camera.cpp

```diff
@@ -1138,7 +1146,6 @@ PSDKWrapper::camera_get_laser_ranging_info_cb(
   }
 }
 
-// TODO(@lidiadltv): Not working. Debug potential issue
 void
 PSDKWrapper::camera_download_file_list_cb(
```

Comment:
This function is massive, maybe it can be split into smaller ones for better readability?